test/OrchardCore.Tests/ContentManagement/ContentElementTests.cs

```diff
@@ -66,4 +66,35 @@ public void Get_WhenCastingConcreteTypeThenBaseType_ReturnNewInstance()
         // contentPart should be returned from cache, so it must be same as actualPart.
         Assert.Same(contentPart, actualPart);
     }
+
+    [Fact]
+    public void Apply_WhenCalledWithNullProperty_SetThePropertyToNull()
+    {
+        var contentItem = new ContentItem();
+
+        contentItem.Apply(new TestContentPart()
+        {
+            Minutes = -15,
```

Comment:
It would be better to use `Theory` to pass the inputs and reduce the duplicate call

Reply:
there is a single parameter here. Why would we change this to `Theory`

Reply:
I will modify the unit test if you don't mind

Reply:
okay